doc/source/development/extending.rst

```diff
@@ -208,13 +208,201 @@ will
 2. call ``result = op(values, ExtensionArray)``
 3. re-box the result in a ``Series``
 
+:class:`~pandas.api.extensions.ExtensionArray` Series Operations Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 0.25.0
+
+In addition to operators like `__mul__` and `__add__`, the pandas Series
+namespace provides a long list of useful operations such as :meth:`Series.round`,
+:meth:`Series.sum`, :meth:`Series.abs`, etc'. Some of these are handled by
+pandas own algorithm implementations (via a dispatch function), while others
+simply call an equivalent numpy function with data from the underlying array.
+In order to support this operations in a new ExtensionArray, you must provide
+an implementation for them.
+
+.. note::
+
+    There is a third category of operation which live on the `pandas`
+    namespace, for example `:meth:pd.concat`. There is an equivalent numpy
+    function `:meth:np.concatenate`, but it is not called by the pandas
+    method. In general, these function should just work with your EA, you do
+    not need to implement more than the general EA interface.
+
+As of 0.25.0, pandas provides its own implementations for the following
+operations: :meth:`Series.any`, :meth:`Series.all`,
+:meth:`Series.min`, :meth:`Series.max`, :meth:`Series.sum`,
+:meth:`Series.mean`, :meth:`Series.median`, :meth:`Series.prod`
+(and its alias :meth:`Series.product`), :meth:`Series.std`,
+:meth:`Series.var`, :meth:`Series.sem`,
+:meth:`Series.kurt`, and :meth:`Series.skew`.
+
+In order to implement any of these functions, your ExtensionArray must include
+an implementation of :meth:`ExtensionArray._reduce`. Once you provide an
+implementation of :meth:`ExtensionArray._reduce` which handles a particular
+method, calling the method on the Series will invoke the implementation
+on your ExtensionArray. All these methods are reduction functions, and
+so are expected to return a scalr value of some type. However it is perfectly
+acceptable to return some instance of an :class:`pandas.api.extensions.ExtensionArray`.
+
+Series operations which are not handled by :meth:`ExtensionArray._reduce`,
+such as :meth:`Series.round`, will generally invoke an equivalent numpy
+function with your extension array as the argument. Pandas only guarantees
+that your array will be passed to a numpy function, it does not dictate
+how your ExtensionArray should interact with numpy's dispatch logic
+in order to achieve its goal, since there are several alternative ways
+of achieving similar results.
+
+However, the details of numpy's dispatch logic are not entirely simple, and
+there nuances which you should be aware of. For that reason, and in order to
+make it easier to create new pandas extensions, we will now cover some of
+possible approaches of dealing with numpy.
+
+The first alternative, and the simplest, is to simply provide an `__array__`
```

Comment:
The base ExtensionArray class already provides a `__array__` method. And if the goal of this section is "compute on your data", I don't think that `__array__` should be included here, since it involves casting to an object-dtype ndarray. I recommend removing this section, or replacing it with "As a stopgap ExtensionArray.__array__ can always be used to convert to a NumPy array of objects, but you probably don't want that because ..."

Reply:
I'm not sure I agree. Casting to object still allows some operations to work for free, for example if the objects themselves provide numpy compatible methods (due to numpy magic). 

I reworded a bit and shortened. See what you think. it starts "For the most basic support..."